docs/source/models/gbd2017/causes/measles.rst

```diff
@@ -34,7 +34,7 @@ measles caused an estimated 2.6 million deaths each year. Despite the
 availability of a vaccine, approximately 110,000 people died from measles in
 2017, mostly children under the age of 5 years. However, due to accelerated
 immunization activities, global measles deaths have decreased 80% during the
-period 2000–2017, from an estimated 545 000 to  110 000, and measles
+period 2000–2017, from an estimated 545,000 to  110,000, and measles
 vaccination prevented an estimated  21.1 million deaths during
 2000–2017 [WHO]_.
```

Comment:
I think that you could add information about the global estimates for vaccine coverage or efficacy if you wanted. I anticipate that the vaccine, and potentially how GBD estimates vaccine coverage, might be important in your model.

Reply:
Good points. When I do the GBD modeling section, I'll mention how they used estimates of vaccine coverage, and I can add some data to the description section if it seems relevant.